editor/src/core/shared/element-template.ts

```diff
@@ -2521,6 +2521,22 @@ export interface ElementInstanceMetadata {
   assignedToProp: string | null
 }
 
+export interface DomElementMetadata {
+  element: Either<string, JSXElementChild>
+  globalFrame: MaybeInfinityCanvasRectangle | null
+  nonRoundedGlobalFrame: MaybeInfinityCanvasRectangle | null
+  specialSizeMeasurements: SpecialSizeMeasurements
+  textContent: string | null
+
+  computedStyle: ComputedStyle | null
+  attributeMetadatada: StyleAttributeMetadata | null // sic fix typo
```

Comment:
Rename refactor surely faster than adding the comment? :)

Reply:
maybe in a separate PR, the diff is _massive_